.github/workflows/main.yml

```diff
@@ -680,5 +680,5 @@ jobs:
         run: |
           go env -w GOFLAGS=-mod=mod
           go install github.com/google/addlicense@2fe3ee94479d08be985a84861de4e6b06a1c7208
-          addlicense -ignore "**/script-empty.sh" -ignore "pkg/testdata/**" -ignore "checks/testdata/**" -l apache -c 'Security Scorecard Authors' -v *
+          addlicense -s -ignore "**/script-empty.sh" -ignore "pkg/testdata/**" -ignore "checks/testdata/**" -l apache -c 'OpenSSF Authors' -v *
```

Comment:
will this break subsequent PRs unless we also update each file's headers in this PR?

Reply:
I think it will

Reply:
> will this break subsequent PRs unless we also update each file's headers in this PR?

From https://github.com/google/addlicense:

> It modifies all source files in place and avoids adding a license header to any file that already has one.

So no, it shouldn't cause failures in existing files, but for PRs that add new files requiring a header, they will error on the license check.

Reply:
Cool, Thanks